dokka-runners/dokka-gradle-plugin/src/main/kotlin/formats/DokkaHtmlPlugin.kt

```diff
@@ -67,6 +67,13 @@ constructor(
     }
 
     private fun DokkaFormatPluginContext.configureHtmlUrlLogging() {
+        project.tasks.withType<LogHtmlPublicationLinkTask>().configureEach {
```

Comment:
Q: why the convention setup moved here?

IMO previously it was more readable. 
I believe it would be nice to have such defaults in one of two places: or during registration of a task, or inside task declaration itself. Otherwise we split this configuration in multiple places for no good reason

Reply:
Fair point, I've simplified the logic.

To explain my intent, generally when setting up tasks I try to use `configureEach {}` to set conventions, so that if a user wants to register a task manually, they don't have to duplicate the config.

When registering a specific instance of the task, maybe there's some conventions that only really make sense for that specific task, and not generally. 

In the case, registering the `index.html` as an input only really makes sense for one task, otherwise all link-logging tasks could depend on the default HTML publication.

But here, it doesn't really matter about sharing conventions for all tasks. If someone wants to register a custom task, they can just set the server URI manually, it's not a big deal. So I reverted it.